tests/test_model.py

```diff
@@ -240,3 +240,16 @@ def test_model_update_component_argument_conditional_false(self):
         # Assert
         self.assertNoAdditions(source_file, self.result)
         self.assertNoDeletions(source_file, self.result)
+
+    def test_model_add_param(self):
+        # Setup
+        source_file = os.path.join(self.data_dir, 'DCMotor.mo')
+        model = Model(source_file)
+
+        # Act
+        model.add_parameter('Real', 'myParam', string_comment='a comment', assigned_value=1.25)
```

Comment:
can you write a test with a string `assigned_value`? I want to make sure that the double-quotes are correctly represented in modelica-land. Thanks!